IMDb.js

```diff
@@ -156,38 +169,40 @@ var testCases = [
 						"creatorType": "scriptwriter"
 					},
 					{
-						"firstName": "Héctor",
-						"lastName": "Alterio",
-						"creatorType": "contributor"
-					},
-					{
 						"firstName": "Norma",
 						"lastName": "Aleandro",
 						"creatorType": "contributor"
 					},
 					{
-						"firstName": "Chunchuna",
-						"lastName": "Villafañe",
+						"firstName": "Héctor",
+						"lastName": "Alterio",
 						"creatorType": "contributor"
 					},
 					{
-						"firstName": "Hugo",
-						"lastName": "Arana",
+						"firstName": "Chunchuna",
+						"lastName": "Villafañe",
 						"creatorType": "contributor"
 					}
 				],
-				"date": "08 Nov 1985",
-				"abstractNote": "After the end of the Dirty War, a high school teacher sets out to find out who the mother of her adopted daughter is.",
-				"extra": "IMDB ID: tt0089276\nIMDB Rating: 7.8 (5,300 votes)",
+				"date": "1985-11-08",
+				"abstractNote": "Directed by Luis Puenzo.  With Norma Aleandro, Héctor Alterio, Chunchuna Villafañe, Hugo Arana. After the end of the Dirty War, a high school teacher sets out to find out who the mother of her adopted daughter is.",
+				"extra": "original-title: La historia oficial\nIMDb ID: tt0089276",
 				"genre": "Drama, History, War",
```

Comment:
From https://aurimasv.github.io/z2csl/typeMap.xml
> genre = [standard] class, type or genre of the item (e.g. “adventure” for an adventure movie, “PhD dissertation” for a PhD thesis)

But I am open for any other suggestion.